bolt/src/main/java/com/slack/api/bolt/context/builtin/EventContext.java

```diff
@@ -15,4 +15,11 @@
 
     private String channelId;
 
+    // X-Slack-Retry-Num: 2 in HTTP Mode
+    // "retry_attempt": 0, in Socket Mode
```

Comment:
@stevengill @mwbrooks 
The names are a bit different here. Do we prefer `retryAttempt`? I personally feel `retryNum` is fine as it's a number.

Reply:
I'd personally prefer to keep the names consistent between the headers and variables. It makes it easier to understand that the two are identical and intended to match. So I'd lean toward `retryNum` as well.